modules/oe_theme_content_project/oe_theme_content_project.post_update.php

```diff
@@ -54,3 +54,23 @@ function oe_theme_content_project_post_update_00002(): void {
   $display = $storage->createFromStorageRecord($display_values);
   $display->save();
 }
+
+/**
+ * Updates the budget field's 'thousand' separator to space.
+ */
+function oe_theme_content_project_post_update_00003(): void {
+  $view_display = EntityViewDisplay::load("node.oe_project.full");
+  $view_display->getComponent('oe_project_budget');
+  $component['settings']['thousand_separator'] = ' ';
+  $view_display->setComponent('oe_project_budget', $component)->save();
+}
+
+/**
+ * Update the formatter plugin for locations field.
+ */
+function oe_theme_content_project_post_update_00004(): void {
+  $view_display = EntityViewDisplay::load('node.oe_project.teaser');
+  $view_display->getComponent('oe_project_locations');
+  $component['type'] = 'oe_theme_helper_address_country_inline';
+  $view_display->setComponent('oe_project_locations', $component)->save();
+}
```

Comment:
By separately I only meant there shouldn't be any loop, not really splitting the update path because it doesn't make sense. The rest of the PR seems fine.